integration_tests/samples/socket_mode/bolt_adapter/aiohttp_async.py

```diff
@@ -0,0 +1,107 @@
+# ------------------
+# Only for running this script here
+import sys
+from os.path import dirname
+
+sys.path.insert(1, f"{dirname(__file__)}/../../../..")
+# ------------------
+
+import asyncio
+import json
+import logging
+import os
+from typing import Optional
+from time import time
+from slack_sdk.socket_mode.response import SocketModeResponse
+from slack_sdk.socket_mode.request import SocketModeRequest
+from slack_sdk.socket_mode.aiohttp import SocketModeClient
+
+from slack_bolt.app.async_app import AsyncApp
+from slack_bolt.request.async_request import AsyncBoltRequest
+from slack_bolt.response import BoltResponse
+
+
+class AsyncSocketModeAdapter:
```

Comment:
don't worry, once we merge these changes into bolt-python library, I will commonize most of the code.